app/assets/javascripts/cable/subscriptions/event.coffee

```diff
@@ -0,0 +1,4 @@
+jQuery ->
+  App.cable.subscriptions.create "EventChannel",
+    received: (data) ->
+      $('#event-list').append '<p>'+data['event']+'<p>'
```

Comment:
I think you mean `prepend` here -- we want to drop it on top rather than on bottom!